openstack/python/tar/slipstream_openstack/OpenStackClientCloud.py

```diff
@@ -263,10 +266,14 @@ def _get_driver(self, user_info):
                                ex_force_auth_url=user_info.get_cloud_endpoint(),
                                ex_force_auth_version=auth_version,
                                ex_force_service_type=user_info.get_cloud('service.type'),
-                               ex_force_service_name=user_info.get_cloud('service.name'),
+                               ex_force_service_name=self._get_service_name(user_info),
                                ex_force_service_region=user_info.get_cloud('service.region'),
                                **kwargs)
 
+    def _get_service_name(self, user_info):
+        service_name = user_info.get_cloud('service.name', '')
+        return service_name if service_name is not None and service_name.strip() != '' else None
+
```

Comment:
I did something like that first but then I changed because if the value is `None` it will failed because `None` doesn't contains the method `strip()`.
For example, it can be `None` if the parameter is not provided to the command line. See [here](https://github.com/slipstream/SlipStreamConnectors/pull/91/files#diff-20d453d9ba5e3550d4eab70e22ef993fR61)